File: atcoder/abc328a_not_too_hard.hs
-- vicfred
-- https://atcoder.jp/contests/abc328/tasks/abc328_a
-- implementation
main :: IO ()
main = do
  [n, x] <- map read . words <$> getLine :: IO [Int]
  s <- map read . words <$> getLine :: IO [Int]
  print . sum . filter (<= x) $ s

File: atcoder/abc328A_not_too_hard.cpp
// vicfred
// https://atcoder.jp/contests/abc328/tasks/abc328_a
// implementation
#include <iostream>

using namespace std;

int main() {
  int N, X;
  cin >> N >> X;
  int ans = 0;
  for (int i = 0; i < N; ++i) {
    int s;
    cin >> s;
    if (s <= X) {
      ans += s;
    }
  }
  cout << ans << endl;
  return 0;
}
